app/lib/failure_reasons.rb

```diff
@@ -29,6 +29,8 @@ class FailureReasons
       "teaching_qualifications_from_ineligible_country",
     TEACHING_QUALIFICATIONS_NOT_AT_REQUIRED_LEVEL =
       "teaching_qualifications_not_at_required_level",
+    WORK_HISTORY_DURATION =
+      WORK_HISTORY_DURATION = "work_history_duration",
```

Comment:
```suggestion
    WORK_HISTORY_DURATION =
      "work_history_duration",
```

The variable only needs to defined once :)